src/Microsoft.ML.Transforms/NormalizerCatalog.cs

```diff
@@ -51,6 +32,106 @@ public static class NormalizationCatalog
             return new NormalizingEstimator(env, mode, InputOutputColumnPair.ConvertToValueTuples(columns));
         }
 
+        /// <summary>
+        /// Normalize (rescale) the column according to the <see cref="NormalizingEstimator.NormalizationMode.MinMax"/> mode.
+        /// It normalizes the data based on the observed minimum and maximum values of the data.
+        /// </summary>
+        /// <param name="catalog">The transform catalog</param>
+        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
+        /// <param name="inputColumnName">Name of the column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
+        /// <param name="maximumExampleCount">Maximum number of examples used to train the normalizer.</param>
+        /// <param name="fixZero">Whether to map zero to zero, preserving sparsity.</param>
+        public static NormalizingEstimator NormalizeMinMax(this TransformsCatalog catalog,
+           string outputColumnName, string inputColumnName = null,
```

Comment:
We used to have example link in the removed API. Can we add it back here?

Reply:
I did add it back.

---
In reply to: [271102840](https://github.com/dotnet/machinelearning/pull/3118#discussion_r271102840) [](ancestors = 271102840)

Reply:
That example link is good for min max normalizer only.